NAMESPACE

```diff
@@ -2,7 +2,5 @@
 
 S3method("$<-",savvy_blazr__sealed)
 S3method("[[<-",savvy_blazr__sealed)
-S3method(print,Person__bundle)
-export(int_times_int)
-export(to_upper)
```

Comment:
`int_times_int`, `to_upper` and the `Person` S3 object are all default implementations from `savvy`. I stripped out that boilerplate code.